getting_started/first-app.rst

```diff
@@ -20,3 +20,15 @@ If everything goes well you should see some front-ends and back-ends showing up
 - http://stilton.yourdomain.com
 - http://cheddar.yourdomain.com
 - http://wensleydale.yourdomain.com
+
+One simply and quick way to access the Traefik UI is to tunneling via ssh to one of the edge nodes with the following command::
+
+    ssh -N -f -L localhost:808x:localhost:808x ubuntu@<edge-floating-ip>
```

Comment:
The second 8080 should be without x, the destination port will always be the same. Also is not so clear what x is, maybe  we could have something like:

```
ssh -N -f -L localhost:8080:localhost:8080 ubuntu@<edge1-floating-ip>
ssh -N -f -L localhost:8081:localhost:8080 ubuntu@<edge2-floating-ip>
...
```